skimage/filters/thresholding.py

```diff
@@ -881,3 +881,42 @@ def threshold_sauvola(image, window_size=15, k=0.2, r=None):
         r = 0.5 * (imax - imin)
     m, s = _mean_std(image, window_size)
     return m * (1 + k * ((s / r) - 1))
+
+
+def apply_hysteresis_threshold(image, low, high):
```

Comment:
@jni in your opinion, is it reasonable to support also `ndarray` type for `low`, `high` arguments (when thresholds are specified by a mask instead of a single value)?